src/puppet-padchat/pure-function-helpers/index.ts

```diff
@@ -12,6 +12,7 @@
  *
  */
 
+export * from './compatible-wei'
```

Comment:
compatible-wei -> compatible-wei-bug

Reply:
This rename is required because that we compatible with wei because it has BUG.

Reply:
you are right, done